// server-launch/config-development.js
/*global module */


module.exports = {
    
    environment: 'development',
    
    
    // shared config items
    data: {
        type: 'mongodb',
        connectionString: 'mongodb://.................'
    },
    
    
    // the paths below will get added to this common root path
    rootPath: '/',
    
    
    // these define configuration for multiple "server apps" that 
    // get attached one common Express server
    apps: [
        
        {
            module: 'server-app-demo-1',
            mainAppClass: 'demo1.MainServerApp',

            // manager specific      
            api: {
                path: 'api/'
            },              
            staticServer: {
                path: ''
            }
        },
        
        {
            module: 'server-app-demo-2',
            mainAppClass: 'demo2.MainServerApp',

            // manager specific                   
            staticServer: {
                path: 'demo2/'
            }
        }
        
    ],
    
    


    the: 'end'
};
